tensorflow/core/kernels/mkl_concat_op.cc

```diff
@@ -626,8 +679,11 @@ class MklConcatOp : public OpKernel {
               src_md =
                   memory::desc(src_dims, MklDnnType<T>(), mkl_common_format);
             }
-
+#ifdef ENABLE_MKLDNN_V1
+            srcs_pd.push_back(memory::desc(src_md));
+#else
             srcs_pd.push_back(memory::primitive_desc(src_md, cpu_engine));
+#endif  // ENABLE_MKLDNN_V1
```

Comment:
We can use
```c++
srcs_pd.push_back(MEMORY_PD_CONSTRUCTOR_2_PARAMS(src_md, cpu_engine));
```

Reply:
I will check and try to address the comments

Reply:
done